src/lib/signature_lib/checked.ml

```diff
@@ -118,11 +121,17 @@ module Schnorr
           val typ : (var, t) Typ.t
 
           val random : unit -> t
```

Comment:
Can you delete this as I think it's no longer use and so we can have a 'proof' that we do deterministic signing?